README.md

```diff
@@ -12,39 +12,45 @@ You need to have *ant*, *git*, *asdf* and *nodeJS* installed.
    Install [asdf](https://asdf-vm.com/guide/getting-started.html).
 2. If you have already used asdf before and have various ant / node versions already installed, you can skip this. Now just make sure to use the project's required nodejs / ant version is running 
 
-        `asdf install nodejs 18.18.2`
-        `asdf install ant 1.10.13`
+        asdf plugin add nodejs
+        asdf install nodejs 18.18.2
+        asdf global nodejs 18.18.2
+   
+        asdf plugin add ant
+        asdf install ant 1.10.13
+        asdf global ant 1.10.13
 
-3. Install gulp
+4. Install gulp
```

Comment:
Ugh, this change wasn't intentional. I think GitHub's web-based editor made the change.